crypto/fipsmodule/rsa/rsa.c

```diff
@@ -1048,6 +1048,12 @@ int RSA_blinding_on(RSA *rsa, BN_CTX *ctx) {
   return 1;
 }
 
+void RSA_blinding_off(RSA *rsa) {
+  if (rsa != NULL) {
+    rsa->flags |= RSA_FLAG_NO_BLINDING;
+  }
```

Comment:
I think `RSA_blinding_on` needs to be updated to invert this, even if it's simply updating `rsa->flags` -- that function is currently a no-op.